runner/src/server/plugins/views.ts

```diff
@@ -78,6 +78,9 @@ export default {
       serviceStartPage: config.serviceStartPage || "#",
       privacyPolicyUrl: config.privacyPolicyUrl || "#",
       phaseTag: config.phaseTag,
+      navigation: request?.auth.isAuthenticated
```

Comment:
maybe `config.authEnabled`? then you can serve a sign in or sign out as well as nothing?